src/static/css/page.css

```
@@ -780,3 +780,28 @@ pre {
 .code-block code {
   width: 100%;
 }
+
+/*
+ * Automaticaly adds language after an anchor, depending on its URL
+ * or the content of its `hreflang` attribute (should only be
+ * used when different lang).
+ *
+
+ */
+[hreflang]::after {
+  content: '(' attr(hreflang) ')';
+  vertical-align: super;
+  font-size: 0.6em;
+}
+
+/*
+ * Fallback for the links that are obviously in English
+ * but that haven’t implemented `hreflang`.
+*/
+html:not([lang="en"]) [href*="/en-US/"]::after,
+html:not([lang="en"]) [href*="/en/"]::after,
+html:not([lang="en"]) [href^="https://en."]::after {
+  content: '(en)';
+  vertical-align: super;
+  font-size: 0.6em;
+}
```

Let's put in other order so the actual `hreflang` has the precedence.

Also let's add an override for `<html lang="en">`/`hreflang="en"` so we can add to English and tackle this in one go (not saying we will add this, but this at least allows us to have that option).

```suggestion
/*
 * Automatically adds language after an anchor if not same language
 *
 * Add for links that are obviously in English
*/
html:not([lang="en"]) [href*="/en-US/"]::after,
html:not([lang="en"]) [href*="/en/"]::after,
html:not([lang="en"]) [href^="https://en."]::after {
  content: '(en)';
  vertical-align: super;
  font-size: 0.6em;
}

/*
 * Add links if an explicit `hreflang` attribute exists
 */
[hreflang]::after {
  content: '(' attr(hreflang) ')';
  vertical-align: super;
  font-size: 0.6em;
}

/*
 * Remove it for English in English pages
 * (allows us to add this to base content to make it easier for translators)
 */
html[lang="en"] [hreflang="en"]::after {
  content: '';
}
```